beprof/example.py

```diff
@@ -0,0 +1,15 @@
+import urllib.request
+from beprof.curve import Curve
+import numpy as np
+
+url = 'http://berkeleyearth.lbl.gov/auto/Global/Complete_TAVG_complete.txt'
+data = urllib.request.urlopen(url)
+
+all = np.genfromtxt(data, skip_header=35, skip_footer=2192)
+print(np.shape(all))
+x = np.array(range(0, 1000))
+y = all[:,3]
+
+example = Curve(np.stack((x, y), axis=1))
```

Comment:
Whole idea behind this library is to deal with data which have X and Y component.
If you have only 1-dimensional data, you can do all such tasks with numpy itself, without much effort.

You should find such dataset which contains X and Y data. X shouldn't be simply a counter running from zero.